assets/src/Components/DocNotice.js

```diff
@@ -0,0 +1,27 @@
+import { Icon } from '@wordpress/components';
+
+const DocNotice = ( { data } ) => {
+	const { text, url } = data;
+
+	if ( ! text ) {
+		return null;
+	}
+
+	return (
+		<div className="doc-notice">
+			{ url ? (
+				<a
```

Comment:
We can use the `<ExternalLink` component from Gutenberg instead of `<a` as it adds the icon before the link as Design team has asked: ![](https://user-images.githubusercontent.com/4193672/234336087-7360e269-587a-4cab-b920-e13424fcc0f6.png)

https://developer.wordpress.org/block-editor/reference-guides/components/external-link/

Reply:
Hey @HardeepAsrani, I thought about that but the Icon is after https://wordpress.github.io/gutenberg/?path=/story/components-externallink--default.

Reply:
Got it, makes sense.